src/vcpkg/base/system.process.cpp

```diff
@@ -246,6 +251,100 @@ namespace vcpkg
 #endif
     }
 
+    Optional<ProcessStat> try_parse_process_stat_file(StringView text, StringView origin)
+    {
+        ParserBase p(text, origin);
+
+        p.match_while(ParserBase::is_ascii_digit); // pid
+
+        p.skip_whitespace();
+        p.require_character('(');
+        auto executable = p.match_until([](char32_t c) { return c == ')'; }).to_string();
```

Comment:
What happens if I put a () in my exe's name?

Reply:
Fixed in https://github.com/microsoft/vcpkg-tool/pull/1049/commits/3807af7340040677395bd1ec4e88402bcd058dd2